src/components/mermaidInputs/InputWithLabelAndValidation/InputWithLabelAndValidation.js

```diff
@@ -53,6 +73,11 @@ const InputWithLabelAndValidation = ({
     />
   )
 
+  const handleCheckboxChange = (checked) => {
+    setIsChecked(checked)
+    handleCheckboxUpdate(checked)
```

Comment:
func passed in from benthic pit inputs component

Reply:
might be worth adding an inline comment somewhere mentioning that an example use case for the checkbox thing is benthic pit.

Reply:
good idea! will add it in